File: CodingTestPlatform/BOJ/1_Bronze/2/10808/10808.py
# 알파벳 개수
# https://www.acmicpc.net/problem/10808

import sys
input = sys.stdin.readline

S = input().rstrip()
A = [0] * 26

for c in S:
    A[ord(c) - 97] += 1
    
print(" ".join(map(str, A)))

File: CodingTestPlatform/BOJ/1_Bronze/2/10808/10808.js
/**
 * 알파벳 개수
 * Bronze2
 * https://www.acmicpc.net/problem/10808
 */

const S = require('fs').readFileSync('/dev/stdin').toString().trim();

const num = 'z'.charCodeAt(0) - 'a'.charCodeAt(0) + 1;
const result = new Array(num).fill(0);

S.split("").forEach((char) => result[char.charCodeAt(0) - 'a'.charCodeAt(0)]++ );

console.log(result.join(" "));